.github/workflows/publish-npm-packages.yml

```diff
@@ -8,17 +8,23 @@ on:
       tag:
         description: 'npm package tag (latest | nightly)'
         required: true
+env:
+  DIR: 'https://raw.githubusercontent.com/ampproject/amphtml/main/build-system/npm-publish'
```

Comment:
Optional: Would it help to split this into...

```yml
REPO: 'https://raw.githubusercontent.com/ampproject/amphtml/main'
DIR: 'build-system/npm-publish'
```
... so that `DIR` can be used for the destination path in the `Get latest scripts` step?